testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -264,6 +264,36 @@ def test_deprecated_warning(self, hole, attrname):
             value = getattr(hole, attrname)
             assert value is hole.results[attrname]
 
+
+@pytest.mark.skipif(executable_not_found('hole'),
+                    reason="Test skipped because HOLE not found")
+class TestOSError():
```

Comment:
```suggestion
class TestOSError:
```
Don't need the `()` here.